### PR TITLE
Fix the properties file for node e2e cri validation.

### PR DESCRIPTION
**Pull request by Random-Liu**
I fixed this locally before, but accidentally missed in the PR. Sorry about that.

This time, I've tried myself, it should work.

@yujuhong

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/33131)

### CODE REVIEW

File: test/e2e_node/jenkins/cri_validation/jenkins-validation.properties [hunk 1]
```diff
@@ -15,4 +15,4 @@ GCE_INSTANCE_METADATA="user-data<${GCI_CLOUD_INIT},gci-docker-version=${DOCKER_V
 CLEANUP=true
 GINKGO_FLAGS='--skip="\[Flaky\]|\[Serial\]"'
 SETUP_NODE=false
-TEST_ARGS='--runtime-integration-type=cri"
+TEST_ARGS='--runtime-integration-type=cri'
```
Comment (yujuhong) [inline test/e2e_node/jenkins/cri_validation/jenkins-validation.properties]: oops. I missed this.

**Comment by Random-Liu** [comment]
Mark `retest-not-required`, because this properties file is not tested in any test.
Mark P2 to match the original PR unblock CRI work.

**Comment by Random-Liu** [comment]
@k8s-bot test this issue #IGNORE.

**Comment by yujuhong** [comment]
@k8s-bot kubemark test this issue: #IGNORE

**Comment by Random-Liu** [comment]
The CRI validation test result is available now https://k8s-testgrid.appspot.com/google-node#kubelet-cri-gce-e2e.

After this got in, we should be able to see bunch of failures and we can start to fix them!! :)

@yujuhong @yifan-gu @feiskyer 

/cc @kubernetes/sig-node

**Comment by yifan-gu** [comment]
@Random-Liu Awesome!

**Pull request merged**